### ISSUE TITLE
Test installation in GW azure

### ISSUE THREAD

**Issue by GINAGC** [k8-proxy/test-installation-documentation#44]

**Comment by MaNe2020** [comment]
As of 12.02. - waiting for testing documentation to be provided

**Comment by samarthdd** [comment]
Waiting for testing documentation to be provided.

**Comment by kerempg** [comment]
- related documentation has ben read through, reviewed. 
- new azure account has been created for testing.
- when testing doc provided, will start processing.

**Comment by Lestat06** [comment]
Waiting for the new version of the documentation to be provided and tested

**Comment by MaNe2020** [comment]
@aduvukace same as per comment above. No date provided when this is expected.

**Comment by svf97** [comment]
@aduvukace this is on hold until v0.4 is sent

**Comment by kerempg** [comment]
1- https://github.com/filetrust/program-icap/pull/288
pull request has been created for FAQ.md

2-  for https://github.com/filetrust/program-icap  which I'm given access, I can not push screenshots via VSCode, it gives the attached error:
![neden (2)](https://user-images.githubusercontent.com/2248132/108159772-c25ce400-70f8-11eb-87ba-71015f6aef3b.png)


is there any way around to overcome the problem of not being able to push  https://github.com/filetrust/program-icap  via VSCode?
or
or any other person, who has write access, can insert the screenshots into the repository.
https://drive.google.com/drive/folders/1BnoEzhLuk0jWZ-sUPuiYaQWz5HfQAsUF?usp=sharing

**Comment by kerempg** [comment]
in progress

**Comment by samarthdd** [comment]
This is still hold until v0.4 is sent

**Comment by GINAGC** [comment]
yes, this is on hold.

**Comment by MaNe2020** [comment]
@aduvukace there is spreadsheet for KT for this task - https://docs.google.com/spreadsheets/d/1HQgbARH_oINw6P1mr2dO8mvSlvOd-gxF9r8eMpWcjaU/edit#gid=0
I suppose once KT is done, this will be resumed. 
Will update again as soon as work starts again on this.

**Comment by GINAGC** [comment]
Task on hold, removing assigned people.
